### ISSUE TITLE
NeuroHub or CONP funding?

### ISSUE THREAD

**Issue by driusan** [datalad/datalad-paper-joss#57]
I don't know details of financing/accounting, so I'm not sure of if they've directly funded development, but I think I've  been to events/hackathons/etc where development was done on DataLad funded by:

1. NeuroHub
2. CONP

Though, to be honest, the boundaries of what's funding for DataLad or `git-annex` or just developers funded from somewhere attending a hackathon and what's significant enough to appear in the funding statement is a little blurry to me.

**Comment by yarikoptic** [comment]
This paper about DataLad core, i.e. https://github.com/datalad/datalad codebase, not necessarily about the DataLad as an entire project.  If any notable code contribution was done with aforementioned financial support , I think we could add it.  Do you have any particular functionality which comes to mind?

**Comment by driusan** [comment]
I don't really know enough about their current involvement to say for sure, I just wanted to flag it in case it becomes political later.

From my own contributions when I was more involved, I think LORIS API/authentication support (and if I remember correctly I had to do some work on generic HTTP header token support to get it working?) would have been in the core repo? Looking at my own PRs to datalad/datalad (where, for some reason I see bug fixes for the LORIS support but not initial the PRs..), there's datalad/datalad#2622 and a few general bug fixes that I had to do to get there which are probably not notable on their own but taken together I think took enough time to cross the threshold for mentioning the funding.

I can try and figure out what grant my working on DataLad in ~2018-2019 would have come from if you agree (it's all a black box to me..)

[edit: I think my funding for working on DataLad at the time would've been from CONP funding]
